docs/D-Modularity/output-functions.md

```diff
@@ -71,28 +75,28 @@ a
 
 The `printf()` function sends data to the output buffer under format control and returns the number of characters sent.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/printf.png)
+![formatted output - printf](https://ict.senecacollege.ca//~ipc144/pages/images/printf.png)
 
-The prototype for the printf() function is:
+The prototype for the `printf()` function is:
 ```c
 int printf(format, argument, ... );
 ```
 
-***format*** is a set of characters enclosed in double-quotes that may consist of any combination of plain characters and conversion specifiers.  The function sends the plain characters as is to the buffer and uses the conversion specifiers to translate each value passed as an argument in the function call.  The ellipsis indicates that the number of arguments can vary.  Each conversion specifier corresponds to one argument. 
+***format*** is a set of characters enclosed in double-quotes that may consist of any combination of plain characters and conversion specifiers. The function sends the plain characters as is to the buffer and uses the conversion specifiers to translate each value passed as an argument in the function call.  The ellipsis indicates that the number of arguments can vary.  Each conversion specifier corresponds to one argument. 
```

Comment:
Since you removed a bunch of the previous extra spaces, there are a few here that you missed. " The ellipsis" and " Each conversion" are the two I see.